vic/drivers/classic/include/vic_driver_classic.h

```diff
@@ -41,7 +41,6 @@ double calc_netshort(double, int, double, double *);
 void check_files(filep_struct *, filenames_struct *);
 FILE  *check_state_file(char *, size_t, size_t, int *);
 void close_files(filep_struct *, out_data_file_struct *, filenames_struct *);
-size_t count_force_vars(FILE *gp);
```

Comment:
Why is this not needed for classic? Wasn't this just introduced? Does that mean that in `vic_classic` the user still needs to specify the number of forcing variables?